packages/newspringchurchapp/src/user-settings/index.js

```diff
@@ -141,7 +142,9 @@ class UserSettings extends PureComponent {
                       <TableView>
                         <Cell>
                           <CellText>
-                            {`App Version: ${getVersion()}.${getBuildNumber()}`}
+                            {`App Version: ${getVersion()}.${getBuildNumber()}${
+                              Config.EXPERIMENTAL ? ' (Experimental)' : ''
```

Comment:
This should make sure that the `EXPERIMENTAL` flag is set to `true`, not just that it exists.

Reply:
ooo you're right